### PR TITLE
Bump go to 1.19.1 and alpine to 3.16

### PR DESCRIPTION
**Pull request by agilob**
Upgrading go attempts to fix https://github.com/grafana/k6/issues/2474

### CODE REVIEW

**Review by codebien — CHANGES_REQUESTED** [review]
Hi @agilob, thanks for your contribution :bow: :tada: 

We would like to keep the `.x` version. I created a similar PR that I'm going to close #2695, but as you can see, there are more files to migrate. Can you migrate all of them, please?

**Comment by agilob** [comment]
updated

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18.x
+          go-version: 1.18x.
```
Comment (codebien) [inline .github/workflows/lint.yml:22]:
```suggestion
          go-version: 1.19.x
```

**Review by codebien — CHANGES_REQUESTED** [review]
One more change required :pray:

**Comment by agilob** [comment]
Sorry!

**Review by codebien — APPROVED** [review]
@agilob thanks for your contribution :clap: 

Does the new Docker image solve your main issue?

**Comment by agilob** [comment]
We'll test it first thing tomorrow morning

**Comment by agilob** [comment]
Tested this internally and it worked on our perf tests. pls merge

**Review by mstoykov — APPROVED** [review]

**Pull request merged**